src/playwright-runner.ts

```diff
@@ -175,7 +175,15 @@ async function getCfg(
     runCfg.sauce = {};
   }
   runCfg.sauce.region = runCfg.sauce.region || 'us-west-1';
-  runCfg.playwrightOutputFolder = path.join(runCfg.assetsDir, 'test-results');
+  runCfg.playwrightOutputFolder =
+    suite.env?.SAUCE_SYNC_WEB_ASSETS?.toLowerCase() === 'true'
```

Comment:
So this will only work if and only if the customer has this env var defined as the string `"true"`? The schema for `env` is a simple object so we won't be able to provide any in-editor context/autocomplete for this value.

Do we want to relax this check and just check for a non empty string?

Reply:
What's your opinion @alexplischke ?

Reply:
But then the customer might want to try with `disabled` to disable it, which can't work as expected.

```
env:
  SAUCE_SYNC_WEB_ASSETS: disabled
```

Reply:
Of course. So what would be the _least_ surprising behaviour?

Reply:
Honestly, I don't feel that strongly one way or the other. If you think enabling if and only if set to `true` is better, let's go with that. We're gonna eventually transition to enabling it by default so this check is gonna be removed soon anyways.

Reply:
Yeah. I think we should set the strict rules with documentation. No vague settings. But of course, this will be removed in the future so I think it's good to leave it like this 🙈

Reply:
Don't feel strongly either way as well. _If you want_ to go the extra mile, you could do a check for `1`, `true`, `enable(d)` and `yes` 😅 Externally we should only document the value "true".